testdata/p4_16_samples_outputs/p4rt_digest_complex-first.p4

```diff
@@ -42,15 +51,26 @@ control MyEC(inout EMPTY a, inout EMPTY b, in psa_egress_input_metadata_t c, ino
     }
 }
 
+struct s2_t_0 {
+    bit<16> f16;
+    bit<32> f;
+}
+
+@name("s3_t") struct s3_t_0 {
```

Comment:
@mbudiu-vmw I don't know if that was discussed at the time of adding generic structs, but this is a bit problematic. It's easy to shoot yourself in the foot as all generated structs will have the same annotation, and the P4Info will actually remove all of the conflicting definitions save for one (so the P4Info will be incorrect).

Reply:
no, the interaction between generic structs and the control plane was never discussed.